account_loan/static/description/index.html

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Why have you changed this file?

Reply:
I just asked for a readme rebuild as it was still linking to 12.0 runbot

![image](https://github.com/OCA/account-financial-tools/assets/55489873/16283f80-b3ef-453b-bb14-18275afefee1)

Reply:
Hi Pedro, this change was made by oca-gen-addon-readme